src/parseSet.java

```diff
@@ -0,0 +1,71 @@
+package src;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+class parseSet {
+
+    private static final String DELIMITER = "[^a-zA-Z0-9_,]+";
```

Comment:
What are you using the delimiter to tokenize and, more to the point, is it doing so properly?

Reply:
Never mind, I checked and I was just being a dumb bitch